### ISSUE TITLE
[BITV] 9.1.4.11/4.1 - The online status indicator in the "settings" menu, represented as a green dot, does not meet the contrast requirements for informative graphics. The contrast ratio is below 3:1. The issue also affects the "away" icon. (3)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41856]
![image](https://github.com/nextcloud/server/assets/6078378/c94b7224-ac3c-4095-98d8-1aad44c2ca7e)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v4-n1

9.1.4.11/6.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v6-n1

9.1.4.11/7.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v7-n1
